### PR TITLE
Initial connections testing

### PR DESCRIPTION
**Pull request by FSFEUP**

### CODE REVIEW

**Review by yvesb04 — APPROVED** [review]

File: src/ros_can/test/test_can.cpp [hunk 1]
```diff
@@ -10,11 +10,77 @@
...
+}
+
+
+/**
+ * @test This test represesnts the case where the steering ot the throttle violates the expected limits (Higher and Lower Value). 
+ * Expected message to not be written to CAN
+ * 
+ */
+TEST_F(RosCanTest, TestOutOfRangeSingleSteeringThrottle) {
+  prepare_control_publish_values(STEERING_UPPER_LIMIT - 1, STEERING_UPPER_LIMIT + 1, 0);
```
Comment (marhcouto) [inline src/ros_can/test/test_can.cpp:82]: Why are we using STEERING_UPPER_LIMIT in the throttle part here?

**Review by marhcouto — CHANGES_REQUESTED** [review]
Just that one detail

File: src/ros_can/test/test_can.cpp (hunk 1 above)
Comment (Davide64-dev) [reply src/ros_can/test/test_can.cpp]: Just reusing the macro, just changed it!

**Comment by Davide64-dev** [comment]
@marhcouto ready for a new review

**Review by marhcouto — APPROVED** [review]

**Pull request merged**